pkg/web/transactions.go

```diff
@@ -648,7 +697,7 @@ func (s *Server) SecureEnvelopeDetail(c *gin.Context) {
 		transactionID uuid.UUID
 		envelopeID    ulid.ULID
 		model         *models.SecureEnvelope
-		out           *api.SecureEnvelope
+		out           any
```

Comment:
Does the change to `any` here mean that the FE will have to check if the return is an `Envelope` or a `SecureEnvelope` to display the envelope detail data?

Reply:
Yes it will - though the front-end does have control of whether or not it is asking for decrypted envelopes or encrypted secure envelopes with the query string. I tried to make the basic details of both the Envelope and the SecureEnvelope (such as ID, EnvelopeID, direction, etc) the same though to reduce friction.